CONTRIBUTING.md

```diff
@@ -197,7 +197,7 @@ $JIRA_URL
 **Documentation**
 
 Documentation for this library may be found at:
-https://docs.mongodb.com/php-library/
+https://mongodb.com/docs/php-library/current/
```

Comment:
@terakilobyte: You missed a few occurrences in README.md. Please add that to this PR.